pkgs/development/libraries/jemalloc/default.nix

```diff
@@ -33,8 +38,19 @@ stdenv.mkDerivation rec {
       url = "https://github.com/jemalloc/jemalloc/commit/4422f88d17404944a312825a1aec96cd9dc6c165.patch";
       hash = "sha256-dunkE7XHzltn5bOb/rSHqzpRniAFuGubBStJeCxh0xo=";
     })
+    ./o3-to-o2.patch
```

Comment:
Perhaps adding a comment for the next maintainers indicating if this patch should be removed in future release or smth like that?